### DISCUSSION TITLE
AOT compilation .NET survey

### DISCUSSION THREAD

**Discussion by nxrighthere** [nxrighthere/UnrealCLR#17] [General]
Please, give your feedback on the AOT compilation in .NET https://www.surveymonkey.com/r/SQV8JQK to enable it in projects like UnrealCLR.

**Comment by nxrighthere** [comment]
Related https://github.com/dotnet/runtime/issues/40430

**Comment by MostHated** [comment]
Done. 👍 I specifically mentioned how great C# projects such as this and many others in all areas of the ecosystem are being held back by the lack of a proper AOT implementation, and them having to come up with their own solutions feels like .Net Framework/.Net Standard all over again. I was being dramatic, but hopefully it makes a difference, lol.

**Comment by nxrighthere** [comment]
Results are [published](https://github.com/dotnet/runtime/issues/41522).

**Comment by hez2010** [comment]
Seems that .NET has started working on Native AOT supported runtime at https://github.com/dotnet/runtimelab/tree/feature/NativeAOT

**Reply by nxrighthere** [reply]
That's great, thanks for sharing!
